src/utils/chem-utils.ts

```diff
@@ -97,6 +220,42 @@ export const chemistryTests: ChemistryTest[] = [
   },
 ];
 
+export type ChemistryValues = Record<ChemTestType, number>;
+export const ChemistryFinalValues: Record<EnvironmentType, ChemistryValues> = {
+  [EnvironmentType.environment1]: {
+    [ChemTestType.airTemperature]: 22,
+    [ChemTestType.waterTemperature]: 12,
+    [ChemTestType.pH]: 7,
+    [ChemTestType.nitrate]: 0,
+    [ChemTestType.turbidity]: 0,
+    [ChemTestType.dissolvedOxygen]: 8
+  },
+  [EnvironmentType.environment2]: {
+    [ChemTestType.airTemperature]: 22,
+    [ChemTestType.waterTemperature]: 12,
+    [ChemTestType.pH]: 6,
+    [ChemTestType.nitrate]: 0,
+    [ChemTestType.turbidity]: 0,
+    [ChemTestType.dissolvedOxygen]: 8
+  },
+  [EnvironmentType.environment3]: {
+    [ChemTestType.airTemperature]: 30,
+    [ChemTestType.waterTemperature]: 22,
+    [ChemTestType.pH]: 5,
+    [ChemTestType.nitrate]: 5,
+    [ChemTestType.turbidity]: 40,
+    [ChemTestType.dissolvedOxygen]: 4
+  },
+  [EnvironmentType.environment4]: {
+    [ChemTestType.airTemperature]: 38,
+    [ChemTestType.waterTemperature]: 24,
+    [ChemTestType.pH]: 9,
+    [ChemTestType.nitrate]: 20,
+    [ChemTestType.turbidity]: 100,
+    [ChemTestType.dissolvedOxygen]: 4
+  }
+};
```

Comment:
So these are the values from the scenario that get fed to the chem lab?  Looks good.